ledger/test-evidence/src/main/scala/com/daml/test/evidence/generator/TestEntryLookup.scala

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.test.evidence.generator
+
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.api.testtool.suites
+import com.daml.test.evidence.generator.TestEntry.{ReliabilityTestEntry, SecurityTestEntry}
+import com.daml.test.evidence.tag.Reliability.{ReliabilityTest, ReliabilityTestSuite}
+import com.daml.test.evidence.tag.Security.{SecurityTest, SecurityTestSuite}
+import org.scalatest.Suite
+import org.scalatest.daml.ScalaTestAdapter
+
+import scala.reflect.ClassTag
+
+object TestEntryLookup {
+
+  private def collectTestEvidence[TT: ClassTag, TS: ClassTag, TE](
+      scalaTestSuites: List[Suite],
+      ledgerApiSuites: List[LedgerTestSuite],
+      testEntry: (String, String, TT, Boolean, Option[TS]) => TE,
```

Comment:
Doesn't tell the caller much about the contents.  Sometimes in this situation I use named types to give an indication.  e.g.  `type SuiteName = String`, Allows `testEntry: (SuiteName, ...`

Reply:
I kept changes here to a minimum as part of this code is shared with canton codebase and this is what they came up with :) 

Let me sync with @soren-da and raise improvements in a separate PR